sys/src/cmd/git/log.c

```diff
@@ -325,7 +325,7 @@ main(int argc, char **argv)
 		sysfatal("chdir: %r");
 
 	gitinit();
-	tmfmtinstall();
+	//tmfmtinstall();
```

Comment:
If we're making this change, then delete this file: git remembers so that we don't have to.

But it may make sense to reach out to Ori and collaborate on a shim library for 9front compat?